Commands/StartCommand.php

```diff
@@ -58,6 +58,10 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $handler->setPhpCgiExecutable($config['php-cgi']);
         $handler->setConcurrentRequestsPerWorker($config['concurrent-requests']);
 
+        if(isset($config['static'])) {
+            $handler->setServingStatic((boolean)$config['static']);
+        }
+
```

Comment:
`loadConfig` already makes sure that `$config['static']` exists, so no need to check at it, also type casting is not necessary since `loadConfig` should do that :+1: